opencraft/urls.py

```diff
@@ -38,10 +40,13 @@
     url(r'^admin/', admin.site.urls),
     url(r'^api/', include('api.urls', namespace='api')),
     url(r'^instance/', include('instance.urls', namespace='instance')),
-    url(r'^registration/', include('registration.urls', namespace='registration')),
+    url(r'^login/$', LoginView.as_view(authentication_form=LoginForm), name='login'),
+    url(r'^logout/$', LogoutView.as_view(next_page='/'), name='logout'),
+    url(r'^registration/$', RedirectView.as_view(url=settings.USER_CONSOLE_FRONTEND_URL), name='registration'),
     url(r'^reports/', include('reports.urls', namespace='reports')),
-    url(r'^email-verification/', include('email_verification.urls', namespace='email-verification')),
     url(r'^favicon\.ico$', RedirectView.as_view(url='/static/img/favicon/favicon.ico', permanent=False)),
+    url(r'^terms/$', RedirectView.as_view(url=settings.OPENCRAFT_TERMS_OF_SERVICE), name='terms'),
```

Comment:
No need to add these URL's here. We can just modify the frontend build settings to update the links. See more information in the other comment.